packages/docusaurus/lib/default-theme/DocBody/styles.css

```diff
@@ -21,14 +21,16 @@
   position: absolute;
   margin-left: -1.25rem;
   padding-right: 0.25rem;
+  opacity: 0;
+  transition: opacity 0.3s;
 }
 
-@media only screen and (min-width: 736px) {
```

Comment:
Why is this removed? This is for the hover behavior only on desktop widths since there's no hover on mobile.

Reply:
dank that was true :| i forget hover doesnt work on mobile